src/locale/zh-cn.js

```diff
@@ -12,16 +12,16 @@ export default moment.defineLocale('zh-cn', {
     weekdaysShort : '周日_周一_周二_周三_周四_周五_周六'.split('_'),
     weekdaysMin : '日_一_二_三_四_五_六'.split('_'),
     longDateFormat : {
-        LT : 'Ah点mm分',
-        LTS : 'Ah点m分s秒',
-        L : 'YYYY-MM-DD',
+        LT : 'HH:mm',
+        LTS : 'HH:mm:ss',
+        L : 'YYYY年MMMD日',
         LL : 'YYYY年MMMD日',
-        LLL : 'YYYY年MMMD日Ah点mm分',
-        LLLL : 'YYYY年MMMD日ddddAh点mm分',
-        l : 'YYYY-MM-DD',
+        LLL : 'YYYY年MMMD日 HH:mm',
```

Comment:
I agree maggiepint. 
In these sites/apps, you can't found `YYYY年MMMD日Ah点mm分`, just because these developer can't found a library like `moment` to format datetime easily.